website/app/components/doc/wcag-list.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="doc-wcag-list">
+  <ul class="doc-wcag-list__ul">
+    {{#each this.criteria as |criterion|}}
+      {{#if (and @criteriaList (includes criterion.id @criteriaList))}}
```

Comment:
suggestion: since you already have the backing class, why not move all the logic inside of it (so the filtering is done there, and here you loop only on the actual criteria that need to be rendered)

Reply:
okay. I did it this way because it was faster for me, & I wanted to get a prototype out there for our eng team to discuss. If the overall approach is acceptable, then I will work on it some more.